lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -574,7 +573,7 @@ WasmBinaryReader::GetWasmToken(WasmBinOp op)
 bool
 WasmBinaryReader::EndOfFunc()
 {
-    return m_funcState.count == m_funcState.size;
+    return m_funcState.count >= m_funcState.size;
```

Comment:
It really should have been >= to begin with. With ==, one can get subtle bug conditions where if m_funcState.count wasn't maintained correctly (e.g., it skipped m_funcState.size or just started out > then m_funcState.size), the decoder and bytecode compiler keeps running. At least that seems to be how we are using m_funcState.count right now. There is another question whether such conditions should be checked separately.